LICENSE

```diff
@@ -0,0 +1,34 @@
+Copyright (c) 2014-2015      UT-Battelle, LLC. All rights reserved.
+Copyright (C) 2014-2015      Mellanox Technologies Ltd. All rights reserved.
+Copyright (C) 2014-2015      The University of Houston System. All rights reserved.
+Copyright (C) 2015           The University of Tennessee and The University 
+                             of Tennessee Research Foundation. All rights reserved.
+Copyright (C) 2016           ARM Ltd. All rights reserved.
+Copyright (c) 2016           Los Alamos National Security, LLC. All rights reserved.
+Copyright (C) 2016-2017      Advanced Micro Devices, Inc.  All rights reserved.
+Copyright (C) 2019           UChicago Argonne, LLC.  All rights reserved.
```

Comment:
Should NVIDIA be listed here somewhere? And if so, should this happen in [the OpenUCX license file]( https://github.com/openucx/ucx/blob/ff93b653fe48d6d4fbbfff7eb66cdee52ed8d9f8/LICENSE#L1-L9 ) as well?

Reply:
I would generally expect that in this repo only the institutions that contributed to it are listed. By checking [contributors to this repo](https://github.com/rapidsai/ucx-py/graphs/contributors), all of them are from NVIDIA, with the exception of @TomAugspurger. At some point, if this gets merged into https://github.com/openucx/ucx, then we would have NVIDIA listed there along all other institutions, but copying over all institutions that contributed to ucx mainline doesn't make much sense to me when it comes to ucx-py, which is not a fork or has no direct relationship apart from being a dependency.

Reply:
Sure. We can change that.

My question is more should NVIDIA also be listed as a contributor in the OpenUCX repo's license file? Thoughts @Akshay-Venkatesh?

Reply:
That's correct, @jakirkham 
Thanks for pointing this out. NVIDIA should be listed. I'll make a PR to add NVIDIA to the main LICENSE file here https://github.com/openucx/ucx/blob/master/LICENSE

Reply:
Thanks Akshay! Feel free to link the PR here. 🙂

Reply:
I've added NVIDIA here and dropped the other entities.

Am unclear on whether Anaconda should be listed here. @mrocklin @TomAugspurger, would you be able to comment on this?

Reply:
I'm not sure what the policy is, but I'm happy to do whatever is easiest for the project (including "waiving" all copyright if that's even necessary / possible).

Reply:
After asking internally was told that only NVIDIA should be listed here. So will leave as-is.